src/plugins/DocsTargetBranch/docs_target_branch.ts

```diff
@@ -0,0 +1,96 @@
+import { PRContext } from "../../types";
+import { Application } from "probot";
+import { filterEventByRepo } from "../../util/filter_event_repo";
+import { filterEventNoBot } from "../../util/filter_event_no_bot";
+import { REPO_HOME_ASSISTANT_IO, ORG_HASS } from "../../const";
+import {
+  extractIssuesOrPullRequestMarkdownLinks,
+  extractPullRequestURLLinks,
+} from "../../util/text_parser";
+
+const NAME = "DocsTargetBranch";
+const SKIP_REPOS = ["brands"];
```

Comment:
Let's also skip the dev docs. 

btw let's use constants for these, in case we rename repos :)

Reply:
https://github.com/home-assistant/probot-home-assistant/blob/master/src/const.ts#L2

Reply:
Moved and changed 👍